app/models/participant_id_change.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class ParticipantIdChange < ApplicationRecord
+  has_paper_trail
```

Comment:
could we postpone adding papertrail for now, we will do a sweep in the end to add it to an audited list of tables if that's ok?